traits/trait_types.py

```diff
@@ -1913,11 +1914,14 @@ class BaseEnum(TraitType):
         The enumeration of all legal values for the trait.  The expected
         signatures are either:
 
-        - a single list, enum.Enum or tuple.  The default value is the first
-          item in the collection.
+        - a single list, enum.Enum, tuple or a collection.  The default value
```

Comment:
I think this could be condensed, since `list`, `tuple` and `enum.Enum` are all cases of a collection. We should also mention that this case excludes strings, bytestrings and bytearrays.